actions/policy.go

```diff
@@ -525,7 +526,7 @@ func (policy *Policy) RemoveProtector(protectorDescriptor string) error {
 func (policy *Policy) Apply(path string) error {
```

Comment:
What about the comparison in `AddProtector()`?

```
        // If the protector is on a different filesystem, we need to add a link  
        // to it on the policy's filesystem.                                     
        if policy.Context.Mount != protector.Context.Mount {                     
```

Reply:
Good point, thanks. Did this not occur in my testing as both policy and protector is set from the same reference in the ceph-csi. But I guess if someone would do a lookup instead between the calls this would fail, so I will change this also